modules/nw-autoscaling-ingress-controller.adoc

```diff
@@ -0,0 +1,222 @@
+// Module included in the following assemblies:
+//
+// * networking/ingress-controller-configuration.adoc
+
+:_content-type: PROCEDURE
+[id="nw-autoscaling-ingress-controller_{context}"]
+= Autoscaling an Ingress Controller
+
+Automatically scale an Ingress Controller to dynamically meet routing performance or availablility requirements such as the requirement to increase throughput. The following procedure provides an example for scaling up the default `IngressController`.
```

Comment:
```suggestion
Automatically scale an Ingress Controller to dynamically meet routing performance or availability requirements such as the requirement to increase throughput. The following procedure provides an example for scaling up the default `IngressController`.
```

Reply:
wow I struggled to even see this typo, thanks